app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -207,7 +207,7 @@ def parse_vapp_template(vapp_template)
       :ems_ref     => uid,
       :name        => vapp_template.name,
       :description => vapp_template.description,
-      :orderable   => false,
+      :orderable   => true,
```

Comment:
@bzwei Yes, vApp templates (orchestration templates) are collected independent of vApps (orchestration stacks). To my knowledge, the vCloud API does not provide any reference between the vApp and the template it was built on. vApps may also be composed from VMs from different vApp templates so I doubt we can provide such a reference.

I've noticed in other providers that they tend to have this reference, but I don't know how to provide it.